samples/android-app/build.gradle

```diff
@@ -0,0 +1,36 @@
+apply plugin: 'android'
```

Comment:
Can't you use the Gradle Script Kotlin version of the DSL/API?

Reply:
Changed in 471d810